tests/kakuro/TestBoardUI.java

```diff
@@ -11,7 +11,8 @@
 import org.junit.Test;
 
 public class TestBoardUI {
-    @Test
+	//The testBoardUIGeneration will test if the board will be created base of the rows and columns defined. If this works, then a bigger board can be produced.
+	@Test
```

Comment:
why the extra space misaligning the method & the decorator?

Reply:
In my codes, its aligned so I am not very sure where the error is. I will work on it

Reply:
the error is that you mix pre-existing 4 space we use everywhere for indent and tabs